test/fixtures/compile-test-data.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Was this script used once to compile test data? It doesn't seem to be run by any other module or command. Also, when running it locally, it complains `localizr: command not found`